documentation/modules/creating-migration-plan-2-8.adoc

```diff
@@ -0,0 +1,70 @@
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-migration-plan-2-8_{context}"]
+ifdef::provider[]
+= Creating a migration plan starting with a source provider
+
+You can create a migration plan based on a source provider, starting on the *Plans for virtualization* page. Note the specific options for migrations from VMware or {rhv-short} providers.
+
+.Procedure
+
+. In the {ocp} web console, click *Plans for virtualization* and then click *Create Plan*.
++
+The *Create migration plan* wizard opens to the *Select source provider* interface.
+. Select the source provider of the VMs you want to migrate.
++
+The *Select virtual machines* interface opens.
+. Select the VMs you want to migrate and click *Next*.
+endif::[]
+
+ifdef::vms[]
+= Creating a migration plan starting with specific VMs
+
+You can create a migration plan based on specific VMs, starting on the *Providers for virtualization* page. Note the specific options for migrations from VMware or {rhv-short} providers.
+
+.Procedure
+
+. In the {ocp} web console, click *Providers for virtualization*.
+. In the row of the appropriate source provider, click *VMs*.
++
+The *Virtual Machines* tab opens.
+. Select the VMs you want to migrate and click *Create migration plan*.
+endif::[]
++
+The *Create migration plan* pane opens. It displays the source provider's name and suggestions for a target provider and namespace, a network map, and a storage map.
+. Enter the *Plan name*.
+. Make any needed changes to the editable items.
+. Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
+. Click *Create migration plan*.
++
+{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. The details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} validates the plan again.
```

Comment:
Please mention the "Settings" with the cold/warm migration.
Or is there a different place for it?

Reply:
@mnecas Good question. Investigating this.